### DISCUSSION TITLE
Inlay_hints at EOL

### DISCUSSION THREAD

**Discussion by LinoWhy** [neovim/neovim#27977] [General]
Can we make the inlay_hints at the end of each line, instead of "inlined" between codes.
With the "inlined" hints, the cursor is jumping back and forth while I'm coding. 

Thanks!

**Comment by prerit714** [comment]
I wish to know as well if there is any way to achieve the same with some config (asking as a newbie)

**Reply by clason** [reply]
Not planned to be configurable (because it violates the specification), but you can of course override the handler and do whatever you want. There are already plugins doing that: https://github.com/chrisgrieser/nvim-lsp-endhints

See https://github.com/neovim/neovim/issues/28261 for discussion.
